smartMeter/ngsild-payloads/consumer.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "urn:ngsi-ld:Consumer:01",
+    "type": "Consumer",
+    "waterConsumption": {
+        "type": "Property",
+        "value": 1500,
+        "unitCode": "LTR",
```

Comment:
OK. What we get from the meter is in fact a number of pulse but it makes more sense to express this in litres. So agreed

Reply:
Ok